deployment/fastcurse/fastcurse.go

```diff
@@ -162,6 +162,7 @@ func applyCurse(cr *CurseRegistry, mcmsRegistry *changesets.MCMSReaderRegistry)
 				}
 				notAlreadyCursedSubjects = append(notAlreadyCursedSubjects, subject)
 			}
+			fmt.Printf("Not already cursed subjects for chain sel %d: %v\n", selector, notAlreadyCursedSubjects)
```

Comment:
NIT: Was this meant to stay, or was this temporary debug? (as it's the only Printf call here).